macros/edr/system/system_utils/get_compiled_code_text.sql

```diff
@@ -0,0 +1,17 @@
+{% macro get_compiled_code_text(node) %}
```

Comment:
cosmetics - if it's still limited to models. let's call it get_compiled_model_code_text?

Reply:
or safe_get_compiled_model_code